### PR TITLE
Add OX team to Teams section

### PR DESCRIPTION
**Pull request by lhansford**
I also did a bunch of cleanup to keep things consistent.

### CODE REVIEW

File: Hiring/Interviewing.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Interviewing is hard for both sides. In less than one hour, you both need to get
 
 * If you have any questions about the scope of the role, the team, or the responsibilities of the position after you've read through those resources be sure to reach out to the Hiring Manager with your questions.
 
-* Open up the [interview kit](https://github.com/fishbrain/docs/tree/main/docs/interviews), and open find the candidates profile in Teamtailor, you’ll be able to view the candidate’s resume, the scorecard you’ll be evaluating the candidate against. 
+* Open up the [interview kit](https://github.com/fishbrain/docs/tree/main/docs/interviews), and open find the candidates profile in Teamtailor, you’ll be able to view the candidate’s resume, the scorecard you’ll be evaluating the candidate against.
```
Comment (bundeeteddee) [inline Hiring/Interviewing.md:19]:
```suggestion
* Open up the [interview kit](https://github.com/fishbrain/docs/tree/main/docs/interviews), and find the candidates profile in Teamtailor, you’ll be able to view the candidate’s resume, the scorecard you’ll be evaluating the candidate against.
```

**Review by bundeeteddee — APPROVED** [review]
Just one comment 👍

File: Hiring/README.md [hunk 2]
```diff
@@ -12,4 +12,6 @@ Collection of Hiring related pages.
 * [Resources in Drive](https://drive.google.com/drive/u/0/folders/1fn_uH7yNdnvrlf_eBP-VEYN8JS5PwSqa)
 * [Job posts](https://drive.google.com/drive/u/0/folders/1iyR2Omvpnz_gQrutSO4M_wyAetbykUMP)
 * [Visas](https://docs.google.com/document/d/1x-5_4Q9NCWCeUWT-lbkRFjpF28F1f7w08Kiynri7UCc/edit)
-* [Interviewers pool](https://docs.google.com/spreadsheets/d/1__H87_iRw3qeFmDnvYhZUId34KYwDhrrOTDXwWApsso/edit#gid=0)
\ No newline at end of file
+* [Interviewers pool](https://docs.google.com/spreadsheets/d/1__H87_iRw3qeFmDnvYhZUId34KYwDhrrOTDXwWApsso/edit#gid=0)
```
Comment (mattias-lundell) [inline Hiring/README.md:15]: this link will fail if fishbrain is not your `gid 0`?

File: Hiring/Teamtailor.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-### Teamtailor
+# Teamtailor
 [Teamtailor](https://www.teamtailor.com/) is Fishbrains ATS (Applicant Tracking System). All Hiring Managers and Interviewers will use Teamtailor to review resumes, provide feedback, communicate with candidates.
 
 ### Adjustments in Teamtailor
```
Comment (mattias-lundell) [inline Hiring/Teamtailor.md:4]: should we remove a `#` here as well?

File: Hiring/Teamtailor.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ There is unfortunately a limit in who can make adjustments to Teamtailors templa
 If you need access to review or interview candidates for a requisition, please reach out to the Recruiting Team through Slack or email. The Recruiting Team will verify with the Hiring Manager and provision access accordingly.
 
 
-### Referrals 
+### Referrals
```
Comment (mattias-lundell) [inline Hiring/Teamtailor.md:13]: https://github.com/fishbrain/engineering-handbook/pull/29/files#r527545837

File: README.md [hunk 5]
```diff
@@ -36,25 +38,29 @@ If you frequently check these channels, you can consider yourself informed.
 
 To scale our infrastructure we have a dev-ops culture including 24/7 support with rotating on-call. Conditions for this is documented in [Google Drive](https://docs.google.com/document/d/1vEP3QwE2EVNqQ6D7FjIIQAwhZe6pXnkfZQvt1vr7OAM/edit).
 
-
 ## Incident management
 
 When we have an incident it is important that we have a uniformed way of acting to the incidents. This will reduce the load on the on-call person and ensure we do the right things at the time of the incident. If there is an incident it will be in the #alert channel in slack and all dialogs about ongoing incidents shall be held there. If there is an outage or another need to get the attention of the on-call person there is a slack command "/pd trigger" that will fire a manual alert. Note that if the alert needs to reach the CTO it is done through PagerDuty and can be done by anyone on-call.
 For all incidents that is impacting our users it is important to share the information within Fishbrain in a transparent way so we can be proactive and honest with our users. Bigger incidents shall be shared in Slack #everybody and minor ones in #support.
 
 ### Security incident
```
Comment (mattias-lundell) [inline README.md:46]: should it be plural?

**Review by mattias-lundell — APPROVED** [review]

**Pull request merged**